### ISSUE TITLE
RMU training

### ISSUE THREAD

**Issue by Qz07** [Qz07/ToxiTIGS#2]
The RMU script does run, however it takes too long (not normal, should take at most an hour, but around 20-30 mins, but it is currently training for 4hrs) and the loss are of high numbers in xxe4 range (also not normal). Need to check the script and correct it.

**Comment by Qz07** [comment]
Gradient accumulation loop is incorrect, it loops through the same batch n times (NOT WHAT WE WANT)
